Cargo.toml

```diff
@@ -138,7 +139,7 @@ clap = { version = "4.4.2", features = ["derive"] }
 dashmap = "5.4.0"
 derive_more = "0.99.17"
 enumflags2 = { version = "0.7.7", features = ["serde"] }
-ethnum = { version = "1.3.2" }
+ethnum = { git = "https://github.com/ariesdevil/ethnum-rs", rev = "4cb05f1" }
```

Comment:
better create pr to merge into upstream later

Reply:
https://github.com/nlordell/ethnum-rs/pull/42

Reply:
We can use the `https://github.com/nlordell/ethnum-rs` with your PR commit.